2022-round-1/deepthi1107/nx_tutorial_script.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+#importing the required libraries
+import networkx as nx
+
+
+# In[2]:
+
+
+#Creating DiGraph
+graph1=nx.DiGraph()
+
+
+# In[3]:
+
+
+#adding node 11,"N1",1,2,3
+graph1.add_node(11)
+graph1.add_node("N1")
+L=[1,2,3]
+graph1.add_nodes_from(L)
```

Comment:
Thanks @deepthi1107!

Could you also add a `tuple` object as a node. Currently there are only `int` and `str` type objects.

Reply:
sure, @MridulS.